src/ext/filer.lisp

```diff
@@ -173,3 +174,9 @@
 
 (define-command filer-select () ()
   (select (back-to-indentation (current-point))))
+
+(define-command filer-project-root () ()
```

Comment:
Would it be okay to overwrite the `M-x filer` command with this command?

Reply:
good point, agree for the better default.